postgres/src/test/java/org/jdbi/v3/postgres/TestPostgresTypes.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.postgres;
+
+import java.util.List;
+
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlCall;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.testing.JdbiRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.postgresql.util.PGobject;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestPostgresTypes {
+
+    @Rule
+    public JdbiRule postgresDbRule = PostgresDbRule.rule();
+
+    private Handle handle;
+
+    @Before
+    public void before() {
+        handle = postgresDbRule.getHandle();
+        handle.useTransaction(h -> {
+            h.execute("drop table if exists postgres_custom_types");
+            h.execute("create table postgres_custom_types(id integer not null, foo text, bar text, created_on timestamp)");
+
+            // create custom type
+            h.execute("drop function if exists get_foo_bars()");
+            h.execute("drop function if exists get_foo_bar(integer)");
```

Comment:
I think each test gets a separate database - these `drop` statements should be unnecessary

Reply:
Even if the same database is used, down below all the statements are using `CREATE OR REPLACE FUNCTION` so these drop statements are definitely redundant.